src/big/big_decimal.cr

```diff
@@ -307,57 +307,91 @@ struct BigDecimal < Number
     self
   end
 
-  # Converts to `Int64`. Truncates anything on the right side of the decimal point.
-  def to_i64
+  # Converts to `BigInt`. Truncates anything on the right side of the decimal point.
+  def to_big_i
     if @value >= 0
-      (@value / TEN ** @scale).to_i64
+      (@value / TEN ** @scale)
     else
-      -(@value.abs / TEN ** @scale).to_i64
+      -(@value.abs / TEN ** @scale)
     end
   end
 
+  # Converts to `Int64`. Truncates anything on the right side of the decimal point.
```

Comment:
The documentation of the unsafe and the safe versions are the same. I think that's pretty confusing.
```suggestion
  # Converts to `Int64`. Truncates anything on the right side of the decimal point.
  # Raises `OverflowError` on overflow.
```
Ditto all below.

Reply:
Yes it should be after OverflowError is introduced